packages/kbn-securitysolution-utils/src/path_validations/index.ts

```diff
@@ -52,20 +52,25 @@ export enum OperatingSystem {
 export type EntryTypes = 'match' | 'wildcard' | 'match_any';
 export type TrustedAppEntryTypes = Extract<EntryTypes, 'match' | 'wildcard'>;
 
-export const validateFilePathInput = ({
+export const validatePotentialWildcardInput = ({
+  fieldName = 'file.path.text',
```

Comment:
should this default value be something different, like `undefined`? Wondering if defaulting to file path can impact other areas.

Reply:
thats a good question. I chose `file.path.text` because the code for this function was originally for that field only, so I'm assuming the usage of this function took that into account. I added the fieldName prop to cover the new cases in the entry builder where we take in other fields that can also uses the `matches` operator.